File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    arr = my_string.split(" ")
    answer = int(arr[0])
    
    for i in range(1, len(arr), 2) :        
        num = int(arr[i + 1])
        if arr[i] == "+":
            answer += num
        elif arr[i] == "-" :
            answer -= num
        
    return answer

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
    const arr = my_string.split(" ");
    var answer = parseInt(arr[0]);
    
    for(let i = 1; i < arr.length; i += 2){
        const num = parseInt(arr[i+1]);
        answer = arr[i] == '+' ? answer + num : answer - num;
    }
    return answer;
}